iranian_cities/models.py

```diff
@@ -1,160 +1,181 @@
 from django.db import models
 from django.utils.translation import gettext_lazy as _
+from iranian_cities.mixins import BaseLocation
 
-
-class Province(models.Model):
-    name = models.CharField(_('Name'), max_length=255)
-    code = models.BigIntegerField(_('Code'))
-
-    def __str__(self):
-        return self.name
-
-    class Meta:
+class Province(BaseLocation):
+    """
+    Represents a province entity within the application.
+    """
+    class Meta(BaseLocation.Meta):
         verbose_name = _('Province')
         verbose_name_plural = _('Provinces')
-        ordering = ('id',)
+        db_table_comment = _("Model representing a province.")
```

Comment:
db_table_comment must not have gettext lazy. remove it.
also add table name to sage_iranian_cities

Reply:
Hi @sepehr-akbarzadeh ,

Thank you for your feedback! 😊

I’ve made the following changes based on your suggestions:

Removed gettext_lazy from db_table_comment to ensure proper handling of database comments.
Added the table name to sage_table_name for clarity and consistency.

Please review the changes and let me know if there are any further adjustments needed. Your feedback is highly 
appreciated!